views/js/pciCreator/ims/likertScaleInteraction/creator/tpl/propertiesForm.tpl

```diff
@@ -0,0 +1,28 @@
+<div class="panel">
+    <label for="level">{{__ "Level"}}</label>
+    <span class="icon-help tooltipstered" data-tooltip="~ .tooltip-content:first" data-tooltip-theme="info"></span>
+    <span class="tooltip-content">{{__ "Scale size"}}</span>
+    <select name="level" class="select2" data-has-search="false">
+        {{#each levels}}
+        <option value="{{@key}}" {{#if selected}}selected="selected"{{/if}}>{{label}}</option>
+        {{/each}}
+    </select>
+</div>
+
+<div class="panel">
+    <label>
+        <input name="icons" type="checkbox" {{#if icons}}checked="checked"{{/if}}/>
+        <span class="icon-checkbox"></span> {{__ "Show icons"}}
+    </label>
+    <span class="icon-help tooltipstered" data-tooltip="~ .tooltip-content:first" data-tooltip-theme="info"></span>
+    <span class="tooltip-content">{{__ "Display icons at each end, or not"}}</span>
+</div>
+
+<div class="panel">
+    <label>
+        <input name="numbers" type="checkbox" {{#if icons}}checked="checked"{{/if}}/>
+        <span class="icon-checkbox"></span> {{__ "Show numbers"}}
```

Comment:
For me number labels are not shown.
(I see that it works on your video.)
![Screenshot_23](https://github.com/oat-sa/extension-tao-itemqti-pci/assets/38751932/a17adda8-016f-4ba5-9f6c-53aadf340a8a)

Reply:
In **Firefox**!
Ok in Chrome.

Reply:
Thanks for checking. I refactored this CSS; no more absolute positioning used in https://github.com/oat-sa/extension-tao-itemqti-pci/pull/394/commits/87eff6291a943e19d379eb374742be29a5315b3f